test/data/bad/crs_nolongervalidtype.result

```diff
@@ -0,0 +1 @@
+[{"message":"crs is not a valid GeoJSON type","line":1}]
```

Comment:
Do we want to start capitalizing the spec language? We haven't done that anywhere else but it might make sense to?